packages/ens-deployments/src/sepolia.ts

```diff
@@ -39,4 +46,8 @@ export default {
       },
     },
   },
+  /**
+   * On the Sepolia "ENS deployment" there is no known subregistry for direct
+   * subnames of 'base.eth' or 'linea.eth'.
```

Comment:
`linea.eth` subregistry has been present on the Sepolia "ENS Deployment":
https://github.com/Consensys/linea-ens/blob/19484c52718dacc129918c19715670e3953119e8/packages/linea-ens-resolver/deployments/sepolia/L1Resolver.json#L2C15-L2C57

Reply:
However, there has been no activity on that L1 resolver contract since its creation:
https://sepolia.etherscan.io/address/0x64884ED06241c059497aEdB2C7A44CcaE6bc7937